docs/discover/search.asciidoc

```diff
@@ -88,6 +88,81 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving queries
+A saved query is a portable collection of query text and filters that you can reuse in <<discover, Discover>>, <<visualize, Visualize>> and <<dashboard, Dashboard>>. Save a query when you want to:
+
+* Retrieve results from the same query at a later time without having to reenter the query text, add the filters or set the time filter again
+* Share your query
+* View the results of the same query in multiple apps
+
+Saved queries don't include information specific to Discover, such as the currently selected columns in the document table, the sort order, and the index pattern. If you want to save your current view of Discover for later retrieval and reuse, create a <<save-open-search, saved search>> instead.
+
+[role="xpack"]
+==== Read only access
+If you have insufficient privileges to save queries, the *Save current query* button isn't visible in the saved query management popover.
+
+[role="screenshot"]
+image::discover/images/saved-query-management-component-readonly-view.png["Example of the saved query management popover with read only access to Discover",width="80%"]
+
+==== Saving a query
+To save the current query text, filters, and time filter:
+
+. Click *#* in the search bar, next to the query text input.
+. Click *Save current query* in the popover.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-all-privileges.png["Example of the saved query management popover with a list of saved queries with write access",width="80%"]
++
+. Enter a name, a description, and then select the filter options that you want to include. By default filters are automatically included, but the time filter is not.
++
+[role="screenshot"]
+image::discover/images/saved-query-save-form-default-filters.png["Example of the saved query management save form with the filters option included and the time filter option excluded",width="80%"]
+. Click *Save*.
+
+==== Loading a query
+To load a saved query into Discover, Dashboard, or Visualize:
```

Comment:
Not sure I understand this.  If you load a query, does it appear in all three apps?

Reply:
No, it only loads in the current app. It is still available to load in the other apps but doesn't do that 'in the background'. If, for example, you're in Discover and you load a saved query, the saved query items will be applied to the current Discover view. If you then navigate to Dashboard, the saved query won't automatically be loaded, you'll have to load it into the Dashboard view manually.